app/controllers/verify/usps_controller.rb

```diff
@@ -11,7 +11,12 @@ def index
     def create
       create_user_event(:usps_mail_sent, current_user)
       idv_session.address_verification_mechanism = :usps
-      redirect_to verify_review_url
+
+      if current_user.decorate.needs_profile_usps_verification?
+        redirect_to account_path
```

Comment:
Code Climate says this is not tested. Can you double check that the specs are performing the correct steps? I would expect at least one of those specs (if not both) to have an expectation that the user ends up on the `account_path`.